integration/raft/channel_participation_test.go

```diff
@@ -414,106 +415,259 @@ var _ = Describe("ChannelParticipation", func() {
 			})
 		})
 
-		It("joins application channels with join-block as follower via channel participation api", func() {
-			orderer1 := network.Orderer("orderer1")
-			orderer2 := network.Orderer("orderer2")
-			orderer3 := network.Orderer("orderer3")
-			orderers := []*nwo.Orderer{orderer1, orderer2}
-			peer := network.Peer("Org1", "peer0")
+		Context("joining application channels with join-block as follower via channel participation api", func() {
+			var (
+				orderer1, orderer2, orderer3 *nwo.Orderer
+				orderers                     []*nwo.Orderer
+				peer                         *nwo.Peer
+				genesisBlock, configBlock    *common.Block
+			)
 
-			By("starting two orderers")
-			for _, o := range orderers {
-				startOrderer(o)
-				cl := channelparticipation.List(network, o)
-				Expect(cl).To(Equal(channelparticipation.ChannelList{}))
-			}
+			BeforeEach(func() {
+				orderer1 = network.Orderer("orderer1")
+				orderer2 = network.Orderer("orderer2")
+				orderer3 = network.Orderer("orderer3")
+				orderers = []*nwo.Orderer{orderer1, orderer2}
+				peer = network.Peer("Org1", "peer0")
+
+				By("starting two orderers")
+				for _, o := range orderers {
+					startOrderer(o)
+					cl := channelparticipation.List(network, o)
+					Expect(cl).To(Equal(channelparticipation.ChannelList{}))
+				}
+
+				genesisBlock = applicationChannelGenesisBlock(network, orderers, []*nwo.Peer{peer}, "participation-trophy")
+				expectedChannelInfoPT := channelparticipation.ChannelInfo{
+					Name:              "participation-trophy",
+					URL:               "/participation/v1/channels/participation-trophy",
+					Status:            "active",
+					ConsensusRelation: "consenter",
+					Height:            1,
+				}
+
+				for _, o := range orderers {
```

Comment:
A nit: here I would add something like
 `By("two out of three orderers join the channel with a genesis block")`

Reply:
Done